content/blog/2023/06/2023-06-20-membership-program.md

```diff
@@ -0,0 +1,150 @@
+enormous.
+
+**A curated protocol benefits individuals, digital rights activists, and
+philanthropic investors who believe in the fundamental right to control and
+privacy in online communication**. As an open standard for real time
+communications, Matrix brings the sovereignty, security and interoperability,
+which are key to having full control over one’s own communication. By
+financially supporting the project, people who are aligned with our beliefs
+allow us to keep fighting for our collective rights, be it by providing secure
+software or persuading regulators to protect their citizens (such as our
+[DMA promotion work](https://matrix.org/blog/2023/03/15/the-dma-stakeholder-workshop-interoperability-between-messaging-services/)
+or our [fight against the Online Safety Bill](https://matrix.org/blog/2021/05/19/how-the-u-ks-online-safety-bill-threatens-matrix)).
+
+**A curated protocol benefits profit-making companies building communication
+products and services.** Decentralisation is a hard problem. End-to-end
+encryption is a hard problem. Decentralised end-to-end encrypted communications
+are a very hard problem. Matrix is doing the heavy lifting in these areas so
+companies can focus on building what they do best: creating great user
+experiences on top of it. By financially supporting the project, organisations
+building on Matrix rest assured it keeps evolving, being patched and staying
+secure. In short it ensures Matrix remains a competitive advantage over other
+products, at a fraction of what it would cost to maintain an in-house solution.
+
+**A curated protocol benefits entire governments, and large parts of the public
+sector**, who have both adopted it widely. Public organisations can not only
+benefit from a sovereign, secure and interoperable solution - they can also
+ensure public money is spent for public good, not shareholder wealth. By joining
+the Matrix Foundation, public sector entities contribute to the stability,
+security and performance of Matrix. Investing into the project ensures that the
+open standard that is supporting their healthcare, defence and public
+administration continues to benefit from innovative open source software
+development.
+
+
+## Stay unbiased & vendor neutral
+
+The Matrix manifesto hasn’t changed. We still believe people should have full
+control over their own communication, not be locked in silos, be able to
+converse securely and privately, and that communication should be available to
+everyone as a free and open, unencumbered, standard and global network.
+
+The Foundation maintains the Spec, which formalises the behaviour expected from
+the various software components of the Matrix ecosystem. We believe it’s
+important for both individuals using the protocol - and organisations building
+products on top of it - to be part of the decision process when it comes to
+shaping the evolution and functionalities of Matrix. So we have designed
```

Comment:
"functionalities" also seems like a very uncommon word, would "features" match the intended meaning?